static/js/protein/chart.ts

```diff
@@ -73,6 +74,13 @@ export interface VarGeneDataPoint {
   upper: number;
 }
 
+type Selectable = Element | EnterElement | Document | Window | SVGLineElement;
+type Datum =
+  | ProteinNumData
```

Comment:
don't need that firs `|`

Reply:
Oh yes!! This bothers me too but I think the linter puts it there automatically so if we took it out every time somebody runs linter on the codebase it would get reinserted.